src/docfx/lib/markdown/MarkdownEngine.cs

```diff
@@ -47,7 +47,7 @@ internal class MarkdownEngine
             _markdownValidationRules = config.MarkdownValidationRules;
             if (!string.IsNullOrEmpty(_markdownValidationRules))
             {
-                using (var stream = restoreFileMap.ReadStream(config.MarkdownValidationRules))
+                using (var stream = fileDownloader.DownloadStream(config.MarkdownValidationRules))
```

Comment:
`_noFetch` in `fileDownloader` is `true` here, it will skip the actual downloading?
Just wondering would it be better to separate the actual downloading and reading from local files?